pkgs/applications/networking/cluster/helm/default.nix

```diff
@@ -9,15 +9,15 @@
 
 buildGoModule rec {
   pname = "kubernetes-helm";
-  version = "3.15.4";
+  version = "3.16.0";
```

Comment:
Would you mind updating it to the `3.16.1` version instead? There's a fix for a regression https://github.com/helm/helm/releases/tag/v3.16.1

Reply:
thanks @Chili-Man :) I've just sent the bump